Library/Formula/rtl-sdr.rb

```diff
@@ -0,0 +1,19 @@
+class RtlSdr < Formula
+  homepage "http://sdr.osmocom.org/trac/wiki/rtl-sdr"
+  url "https://github.com/steve-m/librtlsdr/archive/v0.5.3.tar.gz"
```

Comment:
The official code is the HEAD url, but they do their tagged releases on their official GitHub mirror. I know it's weird. Tell them. :smile_cat: